src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -569,6 +567,80 @@ private void OutputResultsHelper(IEnumerable<CommandInfo> results)
 #endif
         }
 
+        /// <summary>
+        /// Creates the syntax output based on if the command is an alias, script, application or command.
+        /// </summary>
+        /// <param name="command">
+        /// CommandInfo object containing the syntax to be output.
+        /// </param>
+        /// <returns>
+        /// Syntax string cast as a PSObject for outputting.
+        /// </returns>
+        private PSObject GetSyntaxObject(CommandInfo command)
+        {
+            PSObject syntax = PSObject.AsPSObject(command.Syntax);
+
+            if (this.Name != null && !Array.Exists(this.Name, name => name.Equals(command.Name, StringComparison.InvariantCultureIgnoreCase)))
+            {
+                string aliasName = _nameContainsWildcard ? command.Name : this.Name[0];
+                
+                IDictionary<string, AliasInfo> aliasTable = SessionState.Internal.GetAliasTable();
+                foreach (KeyValuePair<string, AliasInfo> tableEntry in aliasTable)
+                {
+                    if (Array.Exists(this.Name, name => name.Equals(tableEntry.Key, StringComparison.InvariantCultureIgnoreCase)) && 
+                        tableEntry.Value.Definition == command.Name)
+                    {
+                        aliasName = tableEntry.Key;
+                        break;
+                    }
+                    else if (_nameContainsWildcard && tableEntry.Value.Definition == command.Name)
+                    {
+                        aliasName = tableEntry.Key;
```

Comment:
You have same result on line 593, you can just || this with line 590

Reply:
Fixed.